src-tauri/src/main.rs

```diff
@@ -90,9 +188,52 @@ async fn main() {
                 });
             }
 
+            // Spawn the Python sidecar on startup
+            println!("[tauri] Creating sidecar...");
+            let sidecar_handle = app.handle().clone();
```

Comment:
I made a second clone of the app handle here because rust was complaining about the old `app_handle` being moved
Unsure if this is how I'm supposed to solve this tho